### PR TITLE
Upgrade to node 8.2.1

### PR DESCRIPTION
**Pull request by kevinsawicki**
Work in progress upgrade to the node ~`v8.2.0-proposal` branch~ `v8.2.1` tag for the chrome 59 upgrade in https://github.com/electron/electron/pull/9946

### CODE REVIEW

File: src/node.cc [hunk 1]
```diff
@@ -2659,9 +2668,9 @@ void FatalException(Isolate* isolate,
 
 void FatalException(Isolate* isolate, const TryCatch& try_catch) {
   HandleScope scope(isolate);
-  if (!try_catch.IsVerbose()) {
+  // if (!try_catch.IsVerbose()) {
```
Comment (kevinsawicki) [inline src/node.cc]: This could be supported if we cherry-pick https://codereview.chromium.org/2840803002 into libcc as a v8 patch.

**Comment by kevinsawicki** [comment]
Switched this to target the node 8.1.4 release instead the 8.2 branch which hasn't been released yet and may or may not ship with v8 6.0

**Comment by addaleax** [comment]
@kevinsawicki I’m nearly 100 % certain Node v8.2.0 won’t ship with V8 6.0, if that helps; and I’m pretty confident we’re going with V8 5.9.

**Comment by kevinsawicki** [comment]
>  I’m nearly 100 % certain Node v8.2.0 won’t ship with V8 6.0, if that helps; and I’m pretty confident we’re going with V8 5.9.

@addaleax oh interesting, I talked with @MylesBorins and I thought he suggested otherwise.

**Comment by MylesBorins** [comment]
I thought we were going to skip 5.9 and wait until 6.0... potentially in a future minor. We likely need to get a free stakeholders together to sync

**Comment by addaleax** [comment]
> I thought we were going to skip 5.9 and wait until 6.0

Sorry, that was not my understanding… I know it has been raised as a possibility in CTC#146, but not as a conclusion? (Especially given that we’re likely able to upgrade to 6.0 in a future minor in the near future). I know we should probably carry that discussion back to the CTC issue but I’m somewhat reluctant to revive it right now…

(In any case, re: the above question … 8.2.0 won’t have V8 6.0 – it will have 5.9 or 5.8.)

**Comment by kevinsawicki** [comment]
> (In any case, re: the above question … 8.2.0 won’t have V8 6.0 – it will have 5.9 or 5.8.)

Do you think 8.2.0 will be released imminently or is it weeks away? Just asking because we are getting close to shipping support for Chrome 59 and would like to go out the door with the latest Node release available at that time.

**Comment by addaleax** [comment]
My goal would be next week, so somewhen between Tuesday and Thursday – if necessary (because we don’t agree it’s a good idea to include it), V8 5.9 is just going to get rebased out of that.

**Comment by kevinsawicki** [comment]
Rebased this to target node 8.2.1

**Pull request closed without merging**